src/components/questionTypes/SC/SCCreationOption.js

```diff
@@ -11,13 +12,25 @@ const propTypes = {
   handleCorrectToggle: PropTypes.func.isRequired,
   handleDelete: PropTypes.func.isRequired,
   name: PropTypes.string.isRequired,
+  toggleEditMode: PropTypes.func.isRequired,
```

Comment:
Please name all callbacks `handle...` as is done for the others (just to stick to a convention)